### PR TITLE
[wip] fix electron ci builds

### PR DESCRIPTION
**Pull request by MatissJanis**

### CODE REVIEW

File: packages/desktop-client/src/browser-preload.browser.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function createBackendWorker() {
     type: 'init',
     version: ACTUAL_VERSION,
     isDev: IS_DEV,
-    hash: process.env.REACT_APP_BACKEND_WORKER_HASH
+    hash: process.env.REACT_APP_BACKEND_WORKER_HASH || 'dev'
```
Comment (MatissJanis) [inline packages/desktop-client/src/browser-preload.browser.js:25]: Running `yarn start` does not set this env variable by default. There probably is a more eloquent way to get this working, so I'm open to suggestions.

But we NEED to fix this to make the local dev server work via `yarn start`.

File: packages/desktop-electron/preload.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 const { ipcRenderer, contextBridge } = require('electron');
+// TODO: remove node module usage from this file
+// see: https://www.electronjs.org/docs/latest/tutorial/sandbox
```
Comment (MatissJanis) [inline packages/desktop-electron/preload.js:3]: This is where I got stuck. Basically we either need to flip over `nodeIntegration` from `false` to `true` (which has security implications) or we need to get rid of these dependencies (move them someplace else?).

File: packages/loot-core/package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "@rschedule/json-tools": "^1.2.0",
     "@rschedule/standard-date-adapter": "^1.2.0",
     "absurd-sql": "0.0.53",
-    "better-sqlite3": "7.4.1",
+    "better-sqlite3": "^8.0.1",
```
Comment (MatissJanis) [inline packages/loot-core/package.json:27]: Necessary to do this if we upgrade `electron-rebuild`. However it then breaks one unit test. Need to investigate what's going on there.

**Comment by MatissJanis** [comment]
Closing this now as I don't plan to continue working on it anytime soon. 

Tracking ticket (open for anyone to pick up again): https://github.com/actualbudget/actual/issues/468

File: packages/loot-core/package.json (hunk 3 above)
Comment (MatissJanis) [reply packages/loot-core/package.json:27]: Fix: https://github.com/actualbudget/actual/pull/714

**Comment by j-f1** [comment]
??? lol @trafico-bot

**Pull request closed without merging**